packages/tom-server/src/user-info-api/services/index.ts

```diff
@@ -23,7 +23,7 @@ class UserInfoService implements IUserInfoService {
       }
 
       if (
-        userInfo.some((u) => u.givenName === undefined || u.sn === undefined)
+        userInfo.some((u) => u.givenName === undefined && u.sn === undefined)
```

Comment:
@jcabannes : the problem here is that "givenName" is an array with ldapts _(because it's a multi-valued attribute)_ and the old ldapjs _(still used as server in tests)_ isn't able to manage this

Reply:
Hi @rezk2ll, it seems that you're the author of this part :wink:. Could you take a look ?